Backend-API/src/main/resources/db/migration/V1__create_table_users.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS users
+(
+    id   SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
```

Comment:
shoould be email and password?

Reply:
Well could be, but now it is, in my opinion, just to set the migration. Than we can create more scripts. And create correct tables with correct values. After this merge, We can create new branch and base on values in User class update table or create new one

Reply:
You can use `SELECT 1 + 1;` if you want to have less invasive query or changes and still want to run something.